samples/05_content_publishers/publish_ogc_services.ipynb

```diff
@@ -32,65 +32,12 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    from arcgis.mapping import WebMap
</pre><p>This import can be removed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2136/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>